sdks/python/apache_beam/io/avroio.py

```diff
@@ -629,3 +629,4 @@ def write_record(self, writer, value):
 
   def close(self, writer):
     writer.flush()
+    writer.fo.close()
```

Comment:
Probably just do a "super(_FastAvroSink, self).close()" since file handle is closed at FileBasedSink.

https://github.com/apache/beam/blob/0ac265b59a679804d105cc05c8a17f0c31402030/sdks/python/apache_beam/io/filebasedsink.py#L144

Reply:
`FileBasedSink.close(writer)` calls writer.close(): https://github.com/apache/beam/blob/ab80cc5f031f7881b688e7fb5b05191fa6b3f80f/sdks/python/apache_beam/io/filebasedsink.py#L152

However FastAvro writer only has `flush()`, but does not have `close()`: https://github.com/fastavro/fastavro/blob/53ed64d95d4f82875d1238b1f76d4d87547e40e1/fastavro/_write_py.py#L336

Reply:
Got it. Thanks.